qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -200,15 +200,15 @@ def new_predefined_dds(scheme=SPIN_ECHO, **kwargs):
     elif scheme == SPIN_ECHO:
         sequence = new_spin_echo_sequence(**kwargs)
     elif scheme == CARR_PURCELL:
-        sequence = new_carr_purcell_sequence(**kwargs)
+        sequence = new_cp_sequence(**kwargs)
```

Comment:
Not sure if I follow the correct convention, but I thought two words should be spelled out, i.e., `carr_purcell` instead of `cp`.

Reply:
Good point. I guess if using `cp` here, I should also change `spin echo` to `se`?
I am just so used to `CP` and `CPMG` sequences and change it without thinking too much.

Reply:
Well, the standard specifies that three or more should be abbreviated (https://github.com/qctrl/.github/blob/master/CONTRIBUTING.md#variable-naming), but I don't think it says anything about not abbreviating fewer.

In this case I think it's just a question of how meaningful the CP abbreviation is. My understanding is that it's a fairly standard abbreviation, so I think abbreviating here is reasonable.

Reply:
SGTM

Reply:
I'd leave "spin echo" as-is too, since I don't think SE is a standard abbreviation (although feel free to correct me if I'm wrong; my evidence is just that a Google of "se dynamical decoupling" doesn't return any obvious spin echo results)

Reply:
@charmasaur thanks for the clarification on abbreviations. That said, when I search for `CPMG` + `NMR` I get what I am looking for, but `CP` + `NMR` doesn't lead `carr purcell` but `cross polarization`.

Reply:
OK, fair enough. If there's another CP sequence that might cause confusion with this then let's leave it expanded.

Reply:
Thanks for the comment, I'll revert the changes of `carr purcell`.